src/utils/utils.js

```diff
@@ -0,0 +1,14 @@
+export function merge(target, ...srcs) {
```

Comment:
I think that This method seems to combine the same keys rather than a normal merge. What about `equalKeyMerge`?

Reply:
I think `merge()` can have various meanings. If I would use this function like "normal merge" as you said, I'd just named it `assign()` like `Object.assign` instead.

Reply:
As you said, the word `merge` is used in a variety of ways to make it a `merge` method, so I thought of the function of merge that I generally think of, so I checked the merge code once again. Since the content of the code is different from the general `merge`, I suggested that it would be better if a name that could more describe the function of the function. I don't have an issue if I think your idea is appropriate. :)

Reply:
Well, I think we both have different impression about the term `merge`.
I just thought in js-env, there's one strong term `assign` that exactly do what normal merge you said, so I thought the term `merge` naturally can have different meanings from it.
Well, as changing function name itself is quite easy, I'd leave it as it is and change it whenever it's needed :)